src/molecule_robotframework/playbooks/roles/robotframework/tasks/pip/Solaris-11.yml

```diff
@@ -1,10 +1,17 @@
 ---
+- name: "Solaris 11.4: Get python mediator"
+  command: pkg mediator -H python3
+  register: python3_results
+
+- set_fact:
+    python3_version: "{{ python3_results.stdout.split()[2] }}"
+
 - name: Ensure python pip is installed.
   become: yes
   pkg5:
      state: present
-     name: pip-35
+     name: pip-{{ python3_version | replace('.','') }}
```

Comment:
I wonder if there is a way to just specify "the latest pip for python3" with a simple pattern (e.g. pip-python3)?